.github/workflows/docker_build.yml

```diff
@@ -2,9 +2,9 @@ name: Workflow to build Docker image
 on:
   workflow_call:
     outputs:
-      image_tar_name:
-        description: "docker image tar name."
-        value: ${{ jobs.build.outputs.image_tar_name }}
+      docker_image_version:
+        description: "docker image version" 
+        value: ${{ jobs.build.outputs.docker_image_version }}
```

Comment:
I suggest to put all meta info you got from docker/metadata.  use name `image_metadata` and it includes all data in here https://github.com/docker/metadata-action#outputs.

Reply:
do you mean use `${{ steps.meta.outputs.tags }}` to get fullname of image? maybe it's not infeasible， like this

Reply:
```
 ##[debug]=> '***/tiffany/tiffany-association:devops'
##[debug]Result: '***/tiffany/tiffany-association:devops'
##[debug]Evaluating: runner.name
##[debug]Evaluating Index:
##[debug]..Evaluating runner:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'name'
##[debug]=> 'gh-runner-cn-2'
##[debug]Result: 'gh-runner-cn-2'
Warning: Skip output 'docker_image_fullname' since it may contain secret.
```

from the logs, it's because we are using REGISTRY which from Github secret. as the tags includes REGISTRY in the url,   it contains secret then. to resolve this, we may need to  use REGISTRY as env not secret. ( registry.wiredcraft.cn)